python/cuml/manifold/t_sne.pyx

```diff
@@ -293,17 +312,17 @@ class TSNE(Base,
             warnings.warn("n_iter = {} might cause TSNE to output wrong "
                           "results. Set it higher.".format(n_iter))
         if metric.lower() != 'euclidean':
-            warnings.warn("TSNE does not support {} but only Euclidean. "
-                          "Will do in the near future.".format(metric))
+            # TODO https://github.com/rapidsai/cuml/issues/862
+            warnings.warn("TSNE does not support {} (only Euclidean).".format(
+                          metric))
             metric = 'euclidean'
         if init.lower() != 'random':
+            # TODO https://github.com/rapidsai/cuml/issues/1029
```

Comment:
This one's a little more up to date since we now have more primitives available (and references the old one)
```suggestion
            # TODO https://github.com/rapidsai/cuml/issues/3458
```